docker-for-mac/release-notes.md

```diff
@@ -13,7 +13,54 @@ This page contains information about the new features, improvements, known issue
 For information about Edge releases, see the [Edge release notes](edge-release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
-## Stable Releases of 2019
+## Docker Desktop Community 2.2.0.0
+
+> [Download](https://hub.docker.com/?overlay=onboarding)
+>
+> You must sign in to Docker Hub to download Docker Desktop.
+
+### Upgrades
+
+- [Docker Compose 1.25.1](https://github.com/docker/compose/releases/tag/1.25.1)
```

Comment:
This has now been bumped to 1.25.2

Reply:
I've update the release note and the URL.